### PR TITLE
Gabriella/update developer guide

### LINKED ISSUE #110: Add introduction for developer guide
add an introduction section in the developer guide

### PR DESCRIPTION
**Pull request by GabriellaTeh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,25 +1,38 @@
 ![PlanusLogo](images/PlanusLogo.png)
 ---
 
-User Guide v1.3
+User Guide v1.3
```
Comment (MarcTzh) [inline docs/UserGuide.md]: remove this space

File: docs/UserGuide.md [hunk 2]
```diff
@@ -36,174 +49,217 @@ PlaNus is a desktop app for **managing tasks, optimized for use via a Command Li
...
+Adds a deadline with title "Return book", and a date and time of "02-01-2020 18:00" to PlaNus.
+* `deadline title:Assignment 1 desc:CS3230 Assignment 1 datetime:01-11-2020 18:00 tag:CS3230`<br>
+Adds a deadline with title “Assignment 1”, description “CS3230 Assignment 1”, and a date and time of “01-01-2020 18:00” with tag "CS3230" to PlaNus.
+<br>
 
-Adds a task of the specified type to PlaNus.
+### 4. Add an event : `event`
 
-Format: `add title:TITLE type:TYPE_OF_TASK [desc:DESCRIPTION] [date:DATE_TIME]`
+Users can add an event to PlaNus.
```
Comment (MarcTzh) [inline docs/UserGuide.md]: Where is the description for what an event is

File: docs/UserGuide.md [hunk 3]
```diff
@@ -36,174 +49,217 @@ PlaNus is a desktop app for **managing tasks, optimized for use via a Command Li
...
-### Add a lesson: `lesson`
+* `event title:Career Talk date:02-01-2020 from:09:00 to:12:30`<br>
+Adds an event with title "Career Talk", and a date "02-01-2020" with start time of "09:00" and end time of "11:30" to PlaNus.
+* `event title:Consultation date:13-10-2020 from:19:00 to:20:30 desc:CS2105 consultation tag:CS2105`<br> and a date "13-10-2020" with start time of "19:00" and end time of "20:30" with tag "CS2105" to PlaNus.
+<br>
+
+### 5. Add a lesson : `lesson`
 
-Adds a lesson to PlaNus.
+Users can add a lesson to PlaNus.
```
Comment (MarcTzh) [inline docs/UserGuide.md]: ^ same as above

File: docs/UserGuide.md [hunk 4]
```diff
@@ -36,174 +49,217 @@ PlaNus is a desktop app for **managing tasks, optimized for use via a Command Li
...
 
 ![list tasks](images/ListTasks.png)
 
 Format: `list`
 
+<br>
+
+### 3. Add a deadline : `deadline`
+
+Users can add a deadline to PlaNus.
```
Comment (MarcTzh) [inline docs/UserGuide.md]: ^

File: src/main/java/seedu/address/model/Planus.java [hunk 5]
```diff
@@ -74,6 +91,14 @@ public void addTask(Task task) {
         tasks.add(task);
     }
 
+    /**
+     * Adds a task to PlaNus.
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/Planus.java]: fix this

**Review by BobbyZhouZijian — APPROVED** [review]
LGTM

**Pull request merged**